docs/README.md

```diff
@@ -15,10 +15,10 @@ This repository contains the ToolJet documentation website code and Markdown sou
 - [Local setup](#local-setup)
 
 ## Feedback
-If you want to give documentation feedback, please join our [Slack Community](https://tooljet.com/slack) and drop us a message.
+If you would like to provide feedback on the documentation, feel free to join our [Slack Community](https://tooljet.com/slack) and send us a message.
```

Comment:
Seems like we have a series of senescences starting with "If", can it be rephrased to?

> We welcome your valuable feedback on the documentation! Please feel free to join our [Community on Slack](https://tooljet.com/slack) and send us a message. We would be delighted to hear from you and assist you with any queries or concerns you may have.